OgreMain/src/WIN32/OgreFileSystemLayer.cpp

```diff
@@ -122,6 +122,17 @@ namespace Ogre
         }
 #endif
 
+        if (char* val = getenv("OGRE_CONFIG_PATHS"))
+        {
+#if OGRE_PLATFORM == OGRE_PLATFORM_WIN32 || OGRE_PLATFORM == OGRE_PLATFORM_WINRT
+            Ogre::StringVector configPaths = Ogre::StringUtil::split(val, ";");
+#else
+            Ogre::StringVector configPaths = Ogre::StringUtil::split(val, ":");
```

Comment:
this file is only compiled on windows

Reply:
to make this cross-platform, you can add after this line:
https://github.com/OGRECave/ogre/blob/a2b9c144e653fd15ebbef0bcb6e37ec8c140cd15/Components/Bites/src/OgreApplicationContextBase.cpp#L30

and call `setConfigPaths`. It will *overwrite* the paths though. Which is consistent with OGRE_PLUGIN_DIR on the other hand.